cognite/neat/exceptions.py

```diff
@@ -102,6 +102,9 @@ def __str__(self) -> str:
         return self.message
 
 
+class NeatValueError(NeatException, ValueError): ...
```

Comment:
It is good practice to add our own exception were it is a user mistake. Then we can later have for example sentry catch all non Neat error which would be bugs in the code.